pkg/sensors/tracing/generickprobe.go

```diff
@@ -758,6 +759,34 @@ func addKprobe(funcName string, instance int, f *v1alpha1.KProbeSpec, in *addKpr
 			argType = gt.GenericTypeFromString(a.Type)
 		}
 
+		if a.ExtractParam != "" && j < api.EventConfigMaxArgs {
+			allArgsBtfChilds[j] = [api.MaxBtfArgChildDepth]api.ConfigBtfArgChild{}
+			spec, err := btf.NewBTF()
+			if err != nil {
+				return errFn(fmt.Errorf("Unable to load BTF file"))
+			}
+
+			partialPath := strings.Split(a.ExtractParam, ".")
+			if len(partialPath) > api.MaxBtfArgChildDepth {
+				return errFn(fmt.Errorf("Exausted research in BTF for type %s. The maximum depth allowed is %d", a.Type, api.MaxBtfArgChildDepth))
+			}
+
+			rootType, err := spec.AnyTypeByName(a.Type)
+			if err != nil {
+				return errFn(fmt.Errorf("Type %s has not been found in BTF", a.Type))
+			}
+			lastChild, err := btf.FindNextBTFType(&allArgsBtfChilds[j], rootType, &partialPath, 0)
+			if err != nil {
+				return errFn(err)
+			}
+
+			if a.OverwriteType != "" {
+				argType = gt.GenericTypeFromString(a.OverwriteType)
+			} else {
+				argType = gt.GenericTypeFromBTF(*lastChild)
+			}
+		}
+
```

Comment:
I think we should put this in function and use it from all the generic probes config code, now we just duplicate the code

Reply:
Yes ! I was thinking about this because it is very ugly this way. But after that, I though maybe there are things that can be mutualized between `generickprobe.go` and `genericlsm.go`. So I put the ugly code like this and later we do refacto.
But if you mention this, I'll start just for this haha